### PR TITLE
Support both comma and dot as number precision separators

### PR DESCRIPTION
**Pull request by siemiatj**
Related to #2278

### CODE REVIEW

**Comment by siemiatj** [comment]
Ready to review. I'm adding additional tests in the meantime.

File: src/components/widget/CustomNumberInput.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+const FLOAT_NUMBER_REGEX_FORMAT = /^-?[0-9]+[.,]?[0-9]*$/;
```
Comment (metas-ts) [inline src/components/widget/CustomNumberInput.js]: I think people will want to write e.g. `,1` for `0.1`...
maybe...if it's easy...and you now even have a test to cover your back...
could you look into that pls 😊 ?

File: src/components/widget/RawWidget.js [hunk 2]
```diff
@@ -129,9 +130,16 @@ export class RawWidget extends Component {
...
   // and send a patch request only if date is changed
   handlePatch = (property, value, id, valueTo, isForce) => {
-    const { handlePatch } = this.props;
+    const { handlePatch, widgetData } = this.props;
     const willPatch = this.willPatch(property, value, valueTo);
 
+    const fieldData = widgetData.find(widget => widget.field === property);
+    let isCostPriceInput = false;
+
+    if (fieldData && fieldData.widgetType === 'CostPrice') {
```
Comment (metas-ts) [inline src/components/widget/RawWidget.js]: do we still need this? AFAIU your PR is about a generic solution..?

Comment (siemiatj) [reply src/components/widget/RawWidget.js]: I don't want to go so far and turn this for all fields. It's gonna be madness, I'm pretty sure.

File: src/components/widget/CustomNumberInput.js (hunk 1 above)
Comment (siemiatj) [reply src/components/widget/CustomNumberInput.js]: We've never supportad that but I might as well just add that.

**Pull request merged**